R/mclosest.R

```diff
@@ -0,0 +1,85 @@
+#'@title Extract closest values in a pairwise manner between two matrices
+#'
+#' @description
+#'
+#' `mclosest` functions calculate the closest rows in a pairwise manner.
```

Comment:
Maybe change to *The `mclosest` function calculates the *closest* rows between to matrices (or data frames) considering pairwise differences between values in columns of `x` and `table`.*

Reply:
Just a suggestion... maybe you find a better description.

Reply:
I think it makes sense, i changed it, if i find better at some point i'll fix it.